docs/microservices/logging-monitoring.md

```diff
@@ -96,19 +102,38 @@ Azure Monitor is a managed service, and configuring an AKS cluster to use Azure
+
+- Logging basic request operations: start/exit time, duration
+- Exceptions thrown
+- Context propagation (e.g. sending a correlation ID across HTTP call boundaries)
+
+Instead, the base libraries and frameworks that handle these operations create rich interrelated Span and Trace data structures and propagate these across contexts. Before OpenTelemetry, these were usually just injected as special log messages or as proprietary data structures specific to the vendor building the monitoring tools. OpenTelemetry also encourages a richer instrumentation data model than a traditional logging-first approach, and the logs are made more useful since log messages are linked to the Traces and Spans where they were generated. This often makes finding logs associated with a specific operation or request very easy.
+
+Many of the Azure SDKs have been instrumented with OpenTelemetry or are in the process of implementing it.
+
+An application developer can add manual instrumentation using the OpenTelemetry SDKs to:
+
+- Add instrumentation where an underlying library does not provide it
+- Enrich the trace context by adding Spans to expose application-specific units of work (e.g. an order loop creating a span for the processing of each order line)
+- Enrich existing Spans with entity keys to enable easier tracing (e.g. add an "OrderID" key/value to the request that processes that order). These are surfaced by the monitoring tools as structured values for querying, filtering and aggregating (without parsing out log message strings or looking for combinations of log message sequences, as was commmon with a logging-first approach.)
+- Propagate trace context by accessing Trace and Span attributes, injecting traceIds into responses and payloads and/or reading traceIds from incoming messages to create Requests and Spans.
+
+Read more about instrumentation and the OpenTelemetry SDKs in the [OpenTelemetry documentation](https://opentelemetry.io/docs/concepts/instrumenting/).
+
 ### Application Insights
 
-For richer log data, we recommend instrumenting your code with Application Insights. This requires adding an Application Insights package to your code and configuring your code to send logging statements to Application Insights. The details depend on the platform, such as .NET, Java, or Node.js. The Application Insights package sends telemetry data to Azure Monitor.
+Application Insights builds on OpenTelemetry, ingesting the rich data provided by instrumented libraries and apps and providing rich visualization and query support. The [Application Insights OpenTelemetry-based instrumentation](azure/azure-monitor/app/opentelemetry-enable.md) for langauges such as .NET, Java, Node.js, or Python handles the data collection.
```

Comment:
> handles the data collection

Data collection is done by OpenTelemetry/its instrumentation libraries. The AzureMonitor exporters for OpenTelemetry, only does one thing - exports the telemetry to Azure backend, and nothing more.

Reply:
Thanks! I'll revise to explain this correctly